CoreScriptsRoot/Modules/VR/Panel3D.lua

```diff
@@ -41,6 +41,10 @@ local cursor = Utility:Create "ImageLabel" {
 	BackgroundTransparency = 1,
 	ZIndex = 10
 }
+local partFolder = Utility:Create "Folder" {
+	Name = "VR",
```

Comment:
This name is short, perhaps we should make it longer to avoid accidental collisions with developer named children (uhg, we have to muck up the devs workspace)